src/renderer/base/renderer.cpp

```diff
@@ -698,55 +681,51 @@ void Renderer::_PaintBufferOutput(_In_ IRenderEngine* const pEngine)
 
-        // Shortcut: don't bother redrawing if the width is 0.
-        if (redraw.Width() > 0)
-        {
-            // Retrieve the text buffer so we can read information out of it.
-            const auto& buffer = _pData->GetTextBuffer();
+        // Retrieve the text buffer so we can read information out of it.
+        const auto& buffer = _pData->GetTextBuffer();
 
-            // Now walk through each row of text that we need to redraw.
-            for (auto row = redraw.Top(); row < redraw.BottomExclusive(); row++)
-            {
-                // Calculate the boundaries of a single line. This is from the left to right edge of the dirty
-                // area in width and exactly 1 tall.
-                const auto screenLine = SMALL_RECT{ redraw.Left(), row, redraw.RightInclusive(), row };
-
-                // Convert the screen coordinates of the line to an equivalent
-                // range of buffer cells, taking line rendition into account.
-                const auto lineRendition = buffer.GetLineRendition(row);
-                const auto bufferLine = Viewport::FromInclusive(ScreenToBufferLine(screenLine, lineRendition));
-
-                // Find where on the screen we should place this line information. This requires us to re-map
-                // the buffer-based origin of the line back onto the screen-based origin of the line.
-                // For example, the screen might say we need to paint line 1 because it is dirty but the viewport
-                // is actually looking at line 26 relative to the buffer. This means that we need line 27 out
-                // of the backing buffer to fill in line 1 of the screen.
-                const auto screenPosition = bufferLine.Origin() - COORD{ 0, view.Top() };
-
-                // Retrieve the cell information iterator limited to just this line we want to redraw.
-                auto it = buffer.GetCellDataAt(bufferLine.Origin(), bufferLine);
-
-                // Calculate if two things are true:
-                // 1. this row wrapped
-                // 2. We're painting the last col of the row.
-                // In that case, set lineWrapped=true for the _PaintBufferOutputHelper call.
-                const auto lineWrapped = (buffer.GetRowByOffset(bufferLine.Origin().Y).WasWrapForced()) &&
-                                         (bufferLine.RightExclusive() == buffer.GetSize().Width());
-
-                // Prepare the appropriate line transform for the current row and viewport offset.
-                LOG_IF_FAILED(pEngine->PrepareLineTransform(lineRendition, screenPosition.Y, view.Left()));
-
-                // Ask the helper to paint through this specific line.
-                _PaintBufferOutputHelper(pEngine, it, screenPosition, lineWrapped);
-            }
+        // Now walk through each row of text that we need to redraw.
+        for (auto row = redraw.Top(); row < redraw.BottomExclusive(); row++)

```

Comment:
This code is just re-indented, because I've moved the `Width() == 0` up (see loop begin).
View it as: https://github.com/microsoft/terminal/pull/10923/files?w=1